platforms/scripts/afu_platform_config

```diff
@@ -804,19 +806,19 @@ def findAfuIfcs(db_path):
     afus = set()
     # Walk all the directories
     for db_dir in db_path:
-        try:
-            # Look for JSON files in each directory
-            for fn in os.listdir(db_dir):
-                if fn.endswith('.json'):
+        # Look for JSON files in each directory
+        for fn in os.listdir(db_dir):
+            if fn.endswith('.json'):
+                try:
                     with open(os.path.join(db_dir, fn)) as f:
                         db = json.load(f)
                         # If it has a module-arguments entry assume the file is
                         # valid
                         if ('module-arguments' in db):
                             afus.add(fn[:-5])
-        except Exception:
-            # Give up on this file or directory if there is any error
-            None
+                except Exception:
+                    # Give up on this file or directory if there is any error
+                    None
```

Comment:
Do we want to give up on a directory if there is an error in just one JSON file? Wouldn't it be better to give up on just that file? Perhaps a little more explicit exception handling could help?

Reply:
It was the old code that gave up on the directory when one error was found. That's fixed in the commit, though I failed to change the comment. I will.